### PR TITLE
Bump wasmvm to 1.0.0-beta6

### PR DESCRIPTION
**Pull request by the-frey**
This should mean we can now test the chain upgrade on `uni-2`, hopefully without a chain halt 🙃

### CODE REVIEW

**Review by giansalex — APPROVED** [review]

File: go.sum [hunk 1]
```diff
@@ -84,6 +84,8 @@ github.com/CosmWasm/wasmd v0.23.0 h1:yOksa/TORt2BtXjUKKqcJ67d6O+Jv2Y6wT1Ekm1Como
 github.com/CosmWasm/wasmd v0.23.0/go.mod h1:kNDnMAQDJyVek9k6SxCNijMCzOROzzUGBRT8r/vr7oY=
 github.com/CosmWasm/wasmvm v1.0.0-beta5 h1:38M8z89LB5cFMYB5vfjewMzz9Pr8TB1QBHdjnrWnkas=
 github.com/CosmWasm/wasmvm v1.0.0-beta5/go.mod h1:mtwKxbmsko1zdwpaKiRkRwxijMmIAtnLaX5/UT2nPFk=
+github.com/CosmWasm/wasmvm v1.0.0-beta6 h1:JYh9Z7qF6h9a2sOOx9hZRwCVS4aFdd+aiwrIXznLn8k=
+github.com/CosmWasm/wasmvm v1.0.0-beta6/go.mod h1:mtwKxbmsko1zdwpaKiRkRwxijMmIAtnLaX5/UT2nPFk=
```
Comment (webmaster128) [inline go.sum:88]: Do the wasmvm v1.0.0-beta5 entries disappear with `go mod tidy`?

**Review by webmaster128 — APPROVED** [review]
LGTM

File: go.sum (hunk 1 above)
Comment (the-frey) [reply go.sum:88]: Good point.

Comment (webmaster128) [reply go.sum:79]: Beautiful. Good to see only one version is in use.

Comment (the-frey) [reply go.sum:79]: @webmaster128 I think your review is now stale, sorry about that - if you're happy to add the tick again 😂 ...

Comment (webmaster128) [reply go.sum:79]: I'm grey because I'm a gues with no permissions here. It was gray all the time.

Comment (the-frey) [reply go.sum:79]: Ah, so that's what that means! TIL

**Review by JakeHartnell — APPROVED** [review]

**Pull request merged**